src/foam/nanos/logger/FileLogger.js

```diff
@@ -53,7 +53,10 @@ try {
 
   handler.setFormatter(new CustomFormatter());
   getLogger().addHandler(handler);
-} catch (IOException e) {}`
+} catch (IOException e) {
+  foam.nanos.logger.Logger logger = (foam.nanos.logger.Logger) getX().get("logger");
```

Comment:
This has to be a system.err else we'll have a loop.  Can't use logging from within logging.

Reply:
fixed